src/compiler/AST/Common/Type.elm

```diff
@@ -3,6 +3,7 @@ module AST.Common.Type exposing
     , Type(..)
     , emptyState
     , getVarId
+    , isNotParametric
```

Comment:
`isNotParametric` seems pretty specific for being an exposed function here :) Can we do `isParametric` and then `not` at the usage site?

Reply:
I wrote it as `isConcrete` first but then worried about that not being clear enough / aligned with the literature terminology. `isParametric` should be better.